airbyte-integrations/connectors/source-iterable/source_iterable/streams.py

```diff
@@ -381,7 +394,7 @@ def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
 
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
         if not self.check_unauthorized_key(response):
-            yield from []
+            return []
```

Comment:
what's the benefit of returning `[]` instead of `yield from []`?

Reply:
after yielding from the empty list generator will execute the following code and yield `records` as well whereas `return` will return the empty list once and finish executing the routine